### ISSUE TITLE
[Feature] Support Clickhouse catalog

### ISSUE THREAD

**Issue by flaming-archer** [apache/gravitino#3888]
### Describe the feature

Clickhouse is very famous and should not lack this metadata.
The community now lacks the clickhouse section.

### Motivation

Our production env requires this part.

### Describe the solution

Maybe I can refer to MySQL to develop it.

**Comment by shaofengshi** [comment]
+1, many users are using Clickhouse; Support Clickhouse is reasonable for Gravitino.

@flaming-archer you already work on it, right?

**Comment by flaming-archer** [comment]
> +1, many users are using Clickhouse; Support Clickhouse is reasonable for Gravitino.
> 
> @flaming-archer you already work on it, right?

Yes, but there have been a lot of things going on in the company lately, so I've been busy with other things. After I relax a bit, I will continue doing this.

**Comment by flaming-archer** [comment]
closed due to https://github.com/apache/gravitino/issues/6017
